backend/packages/Upgrade/src/api/services/MoocletExperimentService.ts

```diff
@@ -136,6 +138,51 @@ export class MoocletExperimentService extends ExperimentService {
     return this.dataSource.transaction((manager) => this.handleDeleteMoocletTransaction(manager, params));
   }
 
+  public async syncImportExperiment(
+    experimentFiles: ExperimentFile[],
+    user: UserDTO,
+    logger: UpgradeLogger
+  ): Promise<ValidatedExperimentError[]> {
+    const { experiments, validatedExperiments } = await this.verifyExperiments(experimentFiles, logger);
+
+    await this.syncAddBulkExperiments(experiments, user, logger);
+    return validatedExperiments;
+  }
+
+  private async syncAddBulkExperiments(
+    experiments: ExperimentDTO[],
+    currentUser: UserDTO,
+    logger: UpgradeLogger
+  ): Promise<Experiment[]> {
+    const createdExperiments = [];
+    for (const experiment of experiments) {
+      try {
+        if ('moocletPolicyParameters' in experiment) {
+          if (!env.mooclets?.enabled) {
+            throw new BadRequestError(
```

Comment:
If we throw an error, even valid experiments will not created.
This should be part of the `verifyExperiment` and the experiment should be skipped.
Users should see this problem before the actual import.